modules/http/src/test/java/org/mule/module/http/utils/RequestUtils.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.http.utils;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+
+public class RequestUtils
```

Comment:
Why have this in a Utils class? You are not reusing this anywhere and even if you did, it would make more sense to create a SocketRequester object which would be a better representation of this.